release_notes/ose_3_2_release_notes.adoc

```diff
@@ -0,0 +1,736 @@
+latest  default/image-from-dockerhub:latest  Less than a second ago ...
+----
++
+When importing, all secrets in your project of those types will be checked. To
+exclude a secret from being a candidate for importing, use the
+`*openshift.io/image.excludeSecret*` annotation set to *true*:
++
+----
+$ oc annotate secret/hub openshift.io/image.excludeSecret=true
+----
+
+- Image stream tags can be set to be automatically imported from remote
+repositories when they change (public or private). {product-title}} will
+periodically query the remote registry and check for updates depending on the
+configuration the administrator sets. By default, images will be checked every
+15 minutes.
++
+To set an image to be imported automatically, use the `--scheduled` flag with
+the `oc tag` command:
++
+----
+$ oc tag --source=docker redis:latest myredis:latest --scheduled
+Tag myredis:latest set to import redis:latest periodically.
+----
++
+You can see which images are being scheduled using `oc describe is myredis`.
++
+Administrators can control whether scheduling is enabled, the polling interval,
+and the rate at which images can be imported via the `*imagePolicyConfig*`
+section in the *_/etc/origin/master/master-config.yaml_* file.
+
+- The integrated Docker registry now supports _image pullthrough_, allowing you to
+tag a remote image into {product-title} and directly pull it from the integrated
+registry as if it were already pushed to the {product-title} registry. If the
+remote registry is configured to use content-offload (sending back a temporary
+redirect URL to the actual binary contents), that value will be passed through
+the {product-title} registry and down to the Docker daemon, avoiding the need to
+proxy the binary contents.
++
+To try pullthrough, tag an image from the DockerHub and then pull it from the
+integrated registry:
++
+----
+$ oc tag --source=docker redis:latest redis:local
+$ oc get is redis
+NAME      DOCKER REPO                     TAGS     UPDATED
+mysql     172.30.1.5:5000/default/redis   local    Less than a second ago
+
+# log into your local docker registry
+$ docker pull 127.30.1.5:5000/default/redis:local
+Using default tag: local
+Trying to pull repository 127.30.1.5:5000/default/redis ... latest: Pulling from 127.30.1.5:5000/default/redis
+47d44cb6f252: Pull complete
+838c1c5c4f83: Pull complete
+5764f0a31317: Pull complete
+60e65a8e4030: Pull complete
+449f8db3c25a: Pull complete
+a6b6487c42f6: Pull complete
+Digest: sha256:c541c66a86b0715bfbb89c5515929268196b642551beccf8fbd452bb00170cde
+Status: Downloaded newer image for 127.30.1.5:5000/default/redis:local
+----
++
+You can use pullthrough with private images; the integrated registry will use
+the same secret you imported the image with to fetch content from the remote
+registry.
+
+- Imported images now contain the size of the image as well as the individual
+layers and size of each layer and report that via `oc describe`.
```

Comment:
Suggest: "The `oc describe` command now reports overall image size as well as ....".